### ISSUE TITLE
Honoured Tattoo of the Tuatara

### ISSUE THREAD

**Issue by GZmanov** [PathOfBuildingCommunity/PathOfBuilding#6786]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the behaviour is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the behaviour in-game?

In game i have 2534 strength.

### What is the behaviour in Path of Building?

In POB i have 2541 strength.

### How to reproduce the issue

1. Add tatto on 10 int node and that he was under action Inertia Crimson Jewel.
I think this is due to the fact that the program counts +4 to all attributes twice.  It turns out 4 strength and also 4 dexterity is transferred into strength. If you remove 4 of all attributes in another place, everything converges.

### Character build code

```shell
eNrtPWtX28a2n-tfocVavV9CYN6ScpNzl3kEaCAQQ5L2fOkaSSNQkSVHkiHkrPPf756RZEvGMjLQ9PTetE1qW7P3zH7O3nseev0_X8exdaOyPEqTNxt4C21YKvHTIEou32x8vHj70tn4n38MXp_J4uo03JlGsX7yj8FPr81nK1Y3Kn6z4dobViGzS1V8qlHR3wHVRCbFlUqTE_lHmh2kwZuN92miNixPJkFU1N_8WOb5ezlWbzZOZCangco2LJn7Kgl25492ALPKrvUz_wqa-YXKjnX3w2mRnqQBNAllnAO-sYyS89S_VsVBlk4nbzb4hnUTqduy0dHJ2enoojG2KGmODWj76fVZLO9Udl7IwsrhrzcbQ2CRvFR7cgx_AzYZTwEV45QyZG_ZTAhss43tlcA70ywvHofhfKJUMAOidMvFpKvtWab2w1D5RXSjdrOo2L2SiT_v0UFbhNid0I-DOJnGRTSJIy2cCopvUbsL5PBeHxihrsYXaSHjvbPzeVtObSpcQrc4osRdDZcWM7jOHj5HxdVODBx-RC8a9ugyiQr1SOCzNMrT5An0NUE7SdydxjEYbq-2IwV2diOLqD2sbtzp2IuSR3FvmCl5GpbqOpJBNM1PVJGpfI5ni3fBnshE7qb5XL54VcszlYE_KVoA6AGAc-Wn4IKaIKtNr9HRcvDOHo-jUPVvuRYxFcC6o3kcHfvnfdutjfhxAxqBt-3X8jydxj1bFnM3RzjD3Yb0pdkUi06Me-prA6OzAmGrZbcmHiUNu3DcFQhbLTHrHuFNqj3Cw9wxfmn_8Gw-EzDuOFvcdmwieCezzq7u8siX8Yn8Go2nY5giLuS1mveHmWB2t8JeXhUJOLguaIpc3MmFt1GmHgW4m8bB4wCvZJp3QjLk8FWmOWsp0MqQwX-lWx8l_gzAXYn3Y5IZ198INVb1oEFGYIo6uvFi1Rtm3k1l0n2igLKzS5VUPd7NgF66ZBXQsVL-1QFEhCNZqAaPudhyKHOcVb581twW9ko-68ZNPq-cWZawmbB-AIsMo1uYuMBuijnHK2fK5bxb3e9y1jGsJ3WOOkH3E5Vd3p1fRSqeU-jafZrXA9yVkx6QhvVN8F4iaPe3jCu9QJczpzvGlFnQb7Zq2NGqZmsOvYwWNcxc-Ihtceoi0m1ABuq9KkYtwJeQDTwECUAzk20AuqIH4Exfe-gpNL8n0B6CvJF5czKjaLWOlc2b6sW6HYeCVAIgArWQ46DujC39Q-dr8Xpgw2ycTufhBbV59-xqyCgBmlQI9tB0XOarIxVM_db073RC7sSQei_QYPMVA4vjZSDdZBeF9K_30uCyN6dMJ-tDnE8nE_C8WvR94XQoAflT1Agju2mfNz4FI276PCJWhRy9O5g3XuwAPxxK9e5lAeJeVyuDoP60zFsv9tBt5nUd5AS85BgmTlNHOUkbE-8Wo90xwFvImnulwKZhz1T8LL2F4V_palm-XmsIGOfW3jmUTCXf7hbxs17Nmx2smN6DaaatoTcNixC9uhkawcEEpMbH0Ti6n-ZfRGPwtHm-JwtpBVV68klmkUwKoptZuZKZf3UMGvNWxrEHXuPNRvNX_W0BENf4X2-bAqf-dDSepFlhxTIvhr6fTpPiUOZXbzZ8X0ghXYo97gYuQyFBiFA_JATxMITumCQu596GAR0pGY9BnrsblvqqEZ7JrLib1Sx1k926rFni58xV1CfUFl7IQm673CaBDGzH445NAbOjJGVhycHXBh2QkxdRYmo3wMI43rDOr9LbYXCj2X-RpnE-61FOJioJ6q8Gx0WmlCUN4_WEoXlhWK2_NAuyRwBGdJ01h8HelVYGiP9FhHDtTY5cQv69-S9IjZnYFIRw880hDtvklAv44gjibmKXIwRfBGS8ziZlhOl2lDqEbhLXsf-9YSUpEAR9UWbjTYwcgCK2jZ1NwXVPggLWTcha4TMmhIpNKmzENjElHFAIijZL5Jg4CEARwgg6YghtmhFQTLi9yVzkoE3IT6kLeG2BDHZnk4BroJsCE0EAmUvpJnddguAz5XQTOxQR0x5vEvAh2EBpDI7ulWL4SG0gRTdxoRNOnU0GCTCMQ9iYbJaEUgilgVyH2XyTO4xBS4eWQGSTMSzMmGwYNhbEAf4BYzZLzhKux0QgW6SAHtuah8wpYWGsDBNbf0bAOWpj4D30jQw2vElBKjDuSl7AjE1gIrCLCRdYhF3MoD0iwDoBYjFjAGox4i4DjmDgDuQbBFAyh2-6HGtcXJMjNCCBcUFbCg1gdEAd1agdF0QDkR-goFgLVHBE2SaMHKCBACCIUxis64CiAI9AhrpbIJEiAeQiIGAT9B5wYQyM3MQCCxA8jFOrgrCheyo0BuhJo3QEB5YgLXnNextwGCWE36AV6AUtOwaFcTD0AxEU0j2AFUMPDgdZMIGEGTMQ79q4FDKoGbKBROYwjddxECgR5YAHFNEGogUoGrAPEejLcUEpwHhdLXGbIaNERhNAYBj4A2PkmtHQH9GkgoyAOVodBOV8wyrAKBurKoRUCybaCEv7_On1x9Gx-fDTVVFM8lfb27e3t1sTWVylofoKceWWn463JwAElv0yv47i-KVGuz2Ef3Yu93f9eHrz7hrnoxv_7vj0-ubbkXu5c46_3dyMnJPL-PRd-s_LI--TOPBeMqbc8Oylexqc0i_7h-8mI8qD89EYjPN3JpyD3cIL0s9HH8LoFN-cvhyN9qcHefTr9ARN3__xz_Gn979enQ0PUvkNtAtfF-pzGBbpuS-np_E7enadHX093PtGz-PIvmUfj95H7i_nQTK-Zc7bw_3JLzsfL70vX37fPcxufr9M_0C3Y_s3u3ifHVzFn68mH0Z88uX4t5Pff_n25dvL4_Heryj59Nv5wT75sDcdBl-n4dHFDffdq4_Xkl4n5Dfvjl9kf_A7b__b72pI3PeZSL6E2a73Ptu5JdFvn16-O7p0vn05OP717myX_EJ2Dz58_uPYP7plw_TD3uXw3dGQfdj9Y3h6PWSXe1-GpyfDyYe9bPjut-HkdvdyJ_111z38enK8e4B3xXT_8PeT_DfiHYjx1S9vT2E8wzdvjAC3awm-Lheu8lKc1TfjB428tVFtWBFMivprFQTcb2ccwbwd62pnrGzeDqOuhsabzBvyznbax8zbOV3tjCNsdIw7KdEWPW9IOinRHrSBsLNj7ezn7dzOdtr1ztuJzn7Bf8ybVVE1yLMpxtc6YM0imMoqHPV3KwCTPkwTyMpUYF3IokhTKw2t4kpZF1OIazIJuH0zl2fFNtnTf59r8z2CH_PaovNt6fsQYfl3UVJsTZLLjfnw9My5Uc3p5UR9pBO6JsKPR-anfPsoOZBjcAoQKuRFJuMLCIni7bOyEx0blCPMtw-jRF2nmawe7RyYTkvifnrBrCK1IOCyIFPLIm9aqLwke7um-_uzQc_8f2c2nF_f9WPBfmwSHRmXqZIO-0ibFzA729-ZFfTnmhez4Vnz8f3l2mHiiB9GYoKqH2wwseAPNphw9wcbTAbwgw0m9_7PZMP8qwmvIPSaKN9UcHRX-sP7tCif6R_rL68N_bmVp1lxoMb5zt3e2flbvYawsKOlqofo1ueqKAtLTZg3G0U2VbqSFMpprH__MJVxpGs7qPnrcblVLkmz8WyVG1Bdpbe6zlxivLib6HrH8Pi4fDKMiwqZ7q6u1ZQ1mWpAVtTIA82PZv_bcD7qXRn7uRl3lECiBwJNqsImjCaKde3J0yPTO__0Qm5QU5THKdB7EKc3usq3gLWErRQRRlcBQ2tPxqTGXNJM8EbrMZ71YHQQOhzmudaBJD-R2fWG9aUkWj_ZKxm4YV2WIfaJKmQAKr19BDF3vq35UqoyfNJb0fSixWclrxOV5zNEOgK8R54p5VUDTUBhy5JXPZL_kpM0_-_cMgPa7kfng2RWktYrY4UuYK9L50xRFjEYXetFXw3Zk6Q53x6gSfPpNDnUhdpHU3UfxxqC08DWaWJp8O3n0ssd8FQQsmq3tquXWR-pmB1o1qBujqFWTI1olm3qfh52AQbno5wAWNQkTSzyNDfwoC6VU0aambL_eSzHj-b4bPL5rKv9kQqbTMd9fUGNxJpheXaz0TtnDblPsZv7SNYxnHLzLoQKGkP-7CRWe0_PYRK_Vk-gcjme_p6vgrcqBN_bdvCfbDvH0M2TTOYegjUmFQC1DOxzzZR7Kp_IKHs0MZ-mcaL3pUUa-BEU1f3_n535v4PiH6p4rIo_OXLcU4XKxtWa56PVZTmWddSlCd9TaVhPpRkGQaTxyvjXsydozlI0_UncT2K90UMlzxddnU2zpnWuK7QF8P6klIA6Q67qkb0nPfvBgDFO0-BcJsF5Ue4YemzEqPEME4PqERQacAtgLYPgu5n8ThrcWfXGtHt2j5_N6nfv_NicHXtsprgAv0a4VIM-mxVUFnqU-JmSuQrqLLY-KvZog38I4xpEz1BZNS6rQvbcbPiUxuBE40f5hRnlS5CsQWwDuid1ojdxUXAik2gyjc1McRZP8yeRuQLdOjWPW30AAYSrEVpNjM8u3rNpfKOyKH-KXt_HsQatc-AVpJUI1qXtLfyJwqfo7SKGNeiqQZ-dql34NV6wxmEM4Q4EOwr3zd0WkCwQVm1JqyjDLXdbQX73dM06v9Vh9lOC17Xju6rY-yxB3iKuNXRpPym3O3_HsCEt8j85P67Ojz8-IluA15sr-kZjFeizVVZMpcYcNFhulrQXRQbBvbIRXbs4dpYBDr9Isz8v8flzDeNxdvEcOqnD9NFilNfXuzZThdEjA7syURjNI7mGtVcfz1VhltPKBTP90di7aXGUTKaFQfdmYxzl_u_eNAz1PQtAZJGZSyT2377d3704-rRf7dtughhG_J5Mx56-OaD8_3yH97kyZ06sfOrl5UeIzSJ1awYCGbeM4lzTFcdykjemEb0iVo08BrgV2Eyrw2h2K8NyXPMG3Zj2vyqYf5NLXSjOItU5rtnzBwZVdqgPd-l97l3Y9PUG3YjKkzK7evO5OaPWwSlzt0Q3Fh3wd5KjH66APRpPZNzZc_X0AU4Uek0UdDUKI1-fD1gtcr2CWrZawZdqlXuFvHcbhr4Uh7kvogtB-bAbuLzzoQu6erqCq-a-iU6ulk-7wfeULztpLx92A8-OPablOt1yLLNWKzC9TxOj5GA0wyguqzAd6PZjNWvSjfC0uFJZdRihC9MJ-Ki6yUrDqTYcdOFptFjBK3NWu4ND-lk3aHn6uIMG_WyFJ2odv-1gaLNNN6ry0GqnI1sFWm7v6-RfdexrhQiqGlIH-8unK5hQH_DsoL96vMJIjP8d3kBCXNbxOsxlodkqh5H6109HY04xPh3N4vnGp2N8G8v8ulPe1dNu8I9lwWUJljJ66YVEG9XTMGjbehqG0WIgMYcdrQ4hjP-CoFbpwHqVA5u1WSHfYprswSRXrJBtT1RmWMsNYU7aWrhKV76U0rUxlgpaXaCxSofLJg8ggrnocEW00w_T7NTpoZKxvuApjZ-G8N5FIU-iEzJtmQTmKoAnEqpvH5hOAFk9stNl4epcpItYX2_XiYQ5eahD--p05nmR6VrBtzQd__Zm46WN6BYiRJ_Jgv9I-eB4Vno1e-gg5tuLgPOZ0Zi6W93wV8BAkb3FXFqeR3ltkqhqY5_-XO_rm-aqvFuorAGZn-sShAFqbrYbSb2I9Mr6-P7ow8f9wcWVsnZkrG1ELyxdqCxLs8GuzvsK6xd1q-LBxyT6MlXW0d4ri3qBG3pS8YBx3-MEYRIKFjiK2NINbaY8THyi_IBQHEjuY8yJJ4jvEk953FMDMxrDgleWwwfmiKwKrCJ9ZeEBBNZx5IMiv7LQ4AXmD2y_H5aiNYmA5cskSQvLU9Z5nN7CD56C_wNpubI-6RO4VhRad-m03IZ0owAgL4BcfUgW8ELgHCW5FSVm8-lEP8MIpKq5mg-OkhAGVlj64jOl96qGaWaR-rGVmt1bSzuo1uO7EOvE9URnwcmlsjL9N2T1W3yjEliV2na2IA-2qM9bmwS8rQ-krQ-j4Wh_8HYag32B4NO7AeRIYwjo7yuBQp7iru_bIkAO48pltoeZZyNfChs5VArkkIAF4E5sFdiCooAJHCDfdrFHAm9RCVpi_1eozxLrDcH_foEdowGzpZtyB4w1v7XPugWBWKeJsg7BnEEupjRklWaQD9jPuqxXLQCVWWWpLQPe9cS6vYpiZV2l5sJIS1plsDnTxrKD7hF9N4muaMG6ZU6X-oC3aeZFQaASC8KEseqWPJK-p2Xp2aGNCfUdjwrbFUwJl0iHCge7CiHqSBcxRALluA53uU-5FMy1CW5L3l4w_5H6Mo0gsLPMYW2rvlOzrR_DGAJSCQmMNQJQ7bJGUzCu0vasKwmGpw1tLAv_SktwnAaQgYOqAD1NOlV-BV4uy6YTGEAvqS3jJ1vKz6NEF1VkNx8x9qnr8xCFPPSpYD4XUilCpPTBaRLhUBb6TkAwYxyaOj6Wnk8IE8phYHT3LKi8g_CVdazvKVhwo8TsRwdFVcllcTXYU18LZbYRhFk6tqot7MbzlVisKLcugAU5-Lixkc0ceD2GPazmy1jKl7L0WBVXpmQbBWpQX6Fwn7G-r5RNkCQMGMk8DyNKWGiHwnEpwtyTxBbA9UBxhjGmIWJBAIxFCiPbcaW_BmN30_G4dDqxkqCm-VU0sXThy9KnXbF1K7MsSrPcmkKjzHon0_Ggye2s5LbMwJ2kCRCg5xXvzmjvO5lNo8FhpPeURv5fOkuIpeI4B0YU1hlQrSvT8KRb1ynocUCkcEAOruKYujAvOCpARDFXebYiSPiBhO-hINhHzA08m8kAY6YYprwtEtbyGaQtkourqNKJerNxder1SuYWafl8ZYJdawKSqT1KUBuDVS4Weaq4VSoZAKF6Qwq4l6z0TCVyfd-HKZ1aBhwikcEL3rKW8uu4jIItneQ9uwE9Tqb2kpn_8G4C8VPkw_DTRA2MwgO5U31VxhI78xjTsz5M9g6SHqIhIyxQIYUJ3hW2RII5zEeBH3rIBmujkivbcRxQAEYCB98Tqv4E_v-VJeymTOngX34mQ2DYv4cBRFqYtIWUz58TC1osCrG0LzP8ShfyFkZofz7WEeYC1CVwswA_jZo6Y4oP1f6RwQpgyEfSGkN5GmjYPg3UF9ZpqdMKKDPt0ZaCl9ncAC_jinbxwwwiZgVP1aT4DwpaVrTgD7YQD7awu43CWWIUx6mE-FIHww_YA_YciT0vtCHW5ZDvOMpnyGeSEaJg0ga9Rx4j1JVgKCH4PFHO_15AiQMzO_1hD31gqXGoZRH1exrRD_PQN048MQ7Q3j9wfTsMA4mUJxybIUG4xK4fSoWkjaWEGI1CLBaELobg14O0AswFAgTmi-BHHPDscQBGS4X6WedOKqt5t3-nBhB355BSQXhwT66B54PYHOVwmPMR4SBSL-TK5STkGEJt3yc6q_F8RiFbpL7nIgL5jccdLCQO8BolIdGUXM3Pck2oztOXNjEs58ue6Lr5wP7ZkrNdFlZ9g2Tl2KzZHZJVaUAajSlj9db2-cELhst6hb7yr1GpWgJ4fyf338rJLFUm_GBtocy5l1cWvRBUCDHsK0ox9ZUbOMzxkCMl823iYx4yzwldGsDv3BWeCgl4EseT3LWVS-22Gom_srRgSihPLi3ghfrcyfDgaHewJz0vntvmeaR3hVpmdac-OQ_zI6SWQEuTvRLbkOJiSLqAwaEnqS1tSMkQ8hzXcwUOFQvAMWPiwQcXh4GHpe1ARsb1vbfcXfS-1WYkzbauWIXMIwsb3fe586cHpvSaGMPR9yxawVRvwalnX-L8bGXaCAG0XhsaLPPiA96y_jJOWED2RBt6xml4qcxpD5kfAEKtny2hj4CHXnrbFDkXbhA6gUsE84OQe0FgwyfquZCoUcwdn3GwJBkw17HhGUY4RDpidYUiGPGgU-SkJXP218qcui3vvXSl4O-lA6xzneZ8OsmAPOsim-piXHaXa0LPfUjhMtWakxlFnvQEZYhgEdpeCIk482yfYdfzOeQoijkuplKwgAaewEIrAXccFDjAovacbOOmtVdZwytr5-Xo5aihCKw1WeMBbcl_LpcqX3iB9VRpwLUOl2ihod4iOBAt2KOkgEQjutQrZFblyTWygUObzRYm7gG2O8v_Lbj9rxD9RQb5JWjlgKwe-I9kQL8mZ1kFKdVLh9a7ZOpfx2rwSQLPDuQ0KWKdZZbeGDJYzKrPemmuejEAhEegW1uOTXirdChCpBh4KCUchQUNoYGNA8dWUrqCIAJzGQoV-C1bl558j3BIqkWoJGQOPnIXQ8umB5vp8ejlwcudliYL2nZpLW2posKJqfOieZCPcae2NZe0KF8aIkKOQBfzVEhV9E_EtmY3Vde96yzY9JCv0vIXjriXeMzLA84Sqio_KZO7anb5f5oAr2jhrLCKhVr5aP8YZm_IH7xYBnqVPYFUZXAGPkY7ugt9Q3adttTWwZjbaR0upFRN6xCSYvDhlBIJyo8D5RLEIa8KkPBCgn2I-CQH-SmHhSHhjm1LooRwEUz-gnqo7eRXWEfTNihve_kXFN3TMce5H4aBgtv3GrY97YX2FfPZ_gVYSrmdVcOZbWgD3gIwe5LvrN00jYP0NrHqt0pYetvwoN6xXAbuURJGJmoyF5P_cOKBuUt_WaBxBrlRkH6FmbR04SM50evaDcVjLiFKxxJO6BHH9yDJpwi5hIuQOA63PYqIVF4YKELsQCK9wuk5zKPwo2sL3umWaUPxPoNT_tx0yqKjDlpBW0GqdEBZzBTkXkAwA2m56sW1_ErntINnP1sV9sELUjru8kTEqg0AL4ipOM7XWXXRqBnCDEj3VFF59XJfQ6HLWNWxrjOVqAIsQ1m4tIv29NEJFyhNfzqFBOJHANPQ_WXV_j2lq_263DUcqwKUp7BGMB221r2kTT0Z2rYvbClsTsEMXL1uKVxwshBhY_2fB_E2UphgDoGL7VAmEPYZI-H9YtdcvxcdK6HLI4VmNCHaQcMLwvoqKWvVJNs1tBf8ftxgXlCjjALyLWrqZsZEyo1MK0Zb25w2q0RVN9k0Cu4vDQnmhT0lVv0uux8BR7-Aw12ixTu6AK_XcE1qPR4cwOxoDcdTiMJb0YPwncCXynYh30M-h_gAIaGvt_aEJ5jUb3jldii5g5iwGQqkHUDELVnoEhyIdvTg0E5NblQE5vW1M5mkk0Lf7DjArWBhrspULLlG8YWwWwpPWzu8OtS-dOgDSpdEJXihYqSkqclWRQh07-HiTMLwkii6LPeasyTaffzQ5UVdXtHCXbFfES0rjoEqRRAyAPOjmwiUXnureWkMVE7dRGVA2d7HRkOulT5AlCjuK9_HvlCO7QmJAuTgIPDsADQfhwHiWEkIcJDv-xI8ukCkO7F0OktjyKSIdVFrONbHGuuQFWb-o0R7zGIexN5eqURXlo_T29IF67pfkVtH4_E00XGFiYohaNAeVe9HdWf7UaNQbwmu9zLWjZYhmJWprfJA5So8zfWLBbin6viqXap4mdSHngIumnff1GXRvUiOYdRz2R8lfpQvCN4mnkscpJcXpACpEgQujoaEMiyw9H3mheD2Qsy9AGHXDanZjaSoy_VyPbY7BY_6OMAnlURp_zJ4n_j271QSJWRppnICztYz6neo5M2dtaPi1hSnqOIu5q4dhtxRLhXIc-zAIUxRSIilDIhCSnAlJfZQoBRhzOGIeD7MdcJ1F3ZZos5kpCHhynot8rPWQOOLjB3rBaR3ejFZJjrYGt9VNjVbWBqZmigjCwvHqJVKzLIQ_Qa2VvVIz2PGnGGub_wOHIp8qzqjVFpG3tgp_xH0A6gKizHEXBazljdPS88U3-lXL8Vmg2KUtepDue4bSFnW36zJvNtMjYFJwd9j2luRRBC6JPw6L9JsDIK4jQej9A7MbmeaXUI0VsxrPDbqrPEQbtPWsh1VniI2RWHocAb5hKtnqJCHELiF0hWSUBGCejMauIEvkMs9COYY9gLQYZ1vfJZZnGaQgMKgB4cw5YByms9N3XZJV_EHcvB2Fr6QpcxT6pYjWnrrK3AinMVW5e4lVysOtau0YV7cNDuL8oXoTJ8NM8m0qQXNJspB32weN6q12tda9XlqXS_SvlDpawgqVc7mbd8rmcGUp812Vk56ickD-c4L4d5Lon5k3uaVqF3LW0dZuZBe6MOdg91smut91dZZrMt5tfVg5HaXSBGk203zcfXb7ATkMTZxlS0kZDV2wAKpmOLSVZRzpULblbYjCGdchsiRHGPsYeU7vu0tJjkdlSr978HCcpheMl86RXTXq-pjg42QoIzYyrWyWiGNMpNGaQrfK5TWRaXGYgVvaP9i6-ZGPr039GMST_1rjbm00Bmgg5flUGgemNQvrKzKWTXgLEYru9hJE33OAOYVJYNyo15utdIzc8xmsLgVZ9GK_wPs6S-vAZBlS3GjKWRCb8GnqQw-g_NarGNRAtqOwsBhduCRUMBkIW3lixAin9CFwIiGZiu-K5SigY9Dmwtl-yjQf0JH9s3-8cBent6LhUiHIaPjCy55VlfqUdHi93fZvSCtxQmz2euBQtaPetWfqavi6VsbMHY8CqF8CNE6wT4DRbUDyVyf2Yrr-B4CINv3iaAhdUPf5cQPiM0cSrjiIW27dIjjl21t-Ay-_PNfu7XBZau2NixUpn5sbXjG-MTuWW_4MI3863zJ_rs9mANbvlYJ6VDHZUhBnsn0QhkVJGTcp0p6kKD6VDtgx5MEwn8HgYMF9-y4gcKS0_uH_b5nzYH3rTl0b0P4e1UanKUeaqQ0a6U1jD0FQWqtApBc6ZeVguOIA70P1FwHON_pwrt3uuitd4PWqgsIjQ6a99vcB-IObSeHngh4GHA7DGy92Yv62AGlwZRLpYSNIFd0JeiZB97QZcyF4NdDnsPBL4ZIBKIzum3ngHoLwEHTG7odOWB1IsXMudhB7axu4XB8oW9TsOS9GR2yOqVrDZ3rAgSRxiGQF7hj3amtpAth6W5ZijBro0mavGwOtMpMNdJFEua7bkzE4PxnRaJ_pefU73PquoSi_aKnWCc8AHHmfRwd67syygv16pc1zd6DOhtrF8T87RQzGPshGGOf1tC7y3NZn3-yGm9fRQ8iUHGxCE_WgC_nicYbbjHpS6e51nWxb7ZG3zNET6C_fp3OM-BYTpB4BoLEMxD0KBzLCMLLEJlHRwFgWXxRcYmxPJ_XeHdx3eNj9Js-A0vJc7FjHYUtX8PRi6FrmKt48gjI2hga74Gmj-uTPpcA-DNoA34GA8PPMA62pio0xPA0t7uOBiy1SfYMHFx_3mnIntgrnVH5NvSWM5q_IL2nM1rJQfpUDpInuoH1BjB7OchT1LiMcR47QVRvnm_KpPEy-p4yWe5c-JNdouhLRPl2-hYR8xfWLxKxBAGlguI2guqn59JM_FyelvTkCWHuAkXlL305wphDFzhS_rQGR0r_0CCdrB6yw2y-MObypyfrInuyQfHnsex1HEy5u64R1buPndLoM0wM9ImukfVUXH3NEWlrQfXTOiHrkgHg55Hgo2OmhiBZP8tpsIzgfiAN_cK07_AaMOIxnOVPnXafSTTsuZzss-Vp9LEZbEPyD4pEr8I1hUj441xc36mFuchZiOOqn3pOLkTYZNHRlz-tlZV26oF4rrrBcxQx2ON0uyFO1ksBmiUr53EK8EzRK3_yZPv06Vqs65Ia_HYe5cPu91hWM-v3hZhbds3rQtIkjC7vvfxD1zjNcfN6h_eG5aVprGRS3d57_30hSm_5O8p3Uv1u9_oFI2dRkkg_XtZ-mqudOCq-VasyD3cwni2El_d4168kEWg59reZSr7dPQJ9eebOHKKb98IRWtaPfqdK-XqvyK9u9H6oI3--6l69r3D3Sun75x8ErIVidleao4T6PRe9wfRWyJK2egVi3T6PkiIaR4Hs1W-DodWVwufmnSd6U3jjhTL1dStnsfSVvpNWZU2NKl-BMmtuz6TQ0d6sZlSvMJhB0QeAZoeP65fN1ICEMPxAf9nsjSRzXVkNone2Lo6QP0QW_LL-4HRP60OZZbA5gMMeHtq65MwYfqbvTJ-JqSfQ2szTOrHIB8cVDzNvneFpPqzVPpu_3KivotbT8UyYkJGSTpA8uozi09B4MrA94wYXX-TU0VE5sFEax2aNrGmAqwFnb_eYdYQwe4AT9daPTlV9vV3PU6-3z2RxdRruTCNze_U_Bv8LrW8jhg==
```

### Screenshots

In game
![in game](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/76125163/dc9cd227-56fc-4f33-8f34-142200cca504)
When reset 1 tattoo
![when reset 1 tattoo](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/76125163/1998538b-cd0f-40a3-aea7-40c8c01b4073)
Without reseting
![1](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/76125163/d4e2a3b2-f709-4e45-aa7a-929abb875ca3)

**Comment by Peechey** [comment]
Fixed by #6712 

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/92683202/dc8b854d-43e9-45fd-a023-43645dd86adb)

**Comment by GZmanov** [comment]
Thank you
